presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -373,6 +378,18 @@ public void updatePartitionStatistics(String databaseName, String tableName, Str
         return delegate.getAllTables(databaseName);
     }
 
+    @Override
+    public List<String> getTablesMatchingParameter(String databaseName, String parameterKey, String parameterValue)
+    {
+        ImmutableList<String> keyTuple = ImmutableList.of(databaseName, parameterKey, parameterValue);
```

Comment:
```suggestion
        List<String> keyTuple = ImmutableList.of(databaseName, parameterKey, parameterValue);
```

Reply:
still needs update (in `Add a metastore method to list tables filtered on a property` commit)